cypress/cypress-integration/integration/practitioners-with-no-cases.cy.ts

```diff
@@ -0,0 +1,60 @@
+import { navigateTo as loginAs } from '../support/pages/maintenance';
+
+describe('Practitioners with no cases', () => {
+  beforeEach(() => {
+    loginAs('petitionsclerk');
+  });
+
+  it('should allow the practitioner to edit their phone number multiple times', () => {
+    loginAs('privatepractitioner4');
+    cy.get('[data-testid="account-menu-button"]').click();
+    cy.get('[data-testid="my-account-link"]').click();
+    cy.get('[data-testid="edit-contact-info"]').click();
+
+    const randomPhoneNumber = generateRandomPhoneNumber();
+    cy.get('[data-testid="phone-number-input"').as('phoneNumberInput');
+    cy.get('@phoneNumberInput').clear();
+    cy.get('@phoneNumberInput').type(randomPhoneNumber);
+
+    cy.get('[data-testid="save-edit-contact"]').click();
+    cy.get('[data-testid="success-alert"]').should('exist');
+
+    cy.get('[data-testid="account-menu-button"]').click();
+    cy.get('[data-testid="my-account-link"]').click();
+    cy.get('[data-testid="contact-info-phone-number"]').should(
+      'contain.text',
+      randomPhoneNumber,
+    );
+
+    cy.get('[data-testid="edit-contact-info"]').click();
+    const anotherRandomPhoneNumber = generateRandomPhoneNumber();
+    cy.get('[data-testid="phone-number-input"').as('phoneNumberInput');
+    cy.get('@phoneNumberInput').clear();
+    cy.get('@phoneNumberInput').type(anotherRandomPhoneNumber);
+
+    cy.get('[data-testid="save-edit-contact"]').click();
+    cy.get('[data-testid="success-alert"]').should('exist');
+
+    cy.get('[data-testid="account-menu-button"]').click();
+    cy.get('[data-testid="my-account-link"]').click();
+    cy.get('[data-testid="contact-info-phone-number"]').should(
+      'contain.text',
+      anotherRandomPhoneNumber,
+    );
+  });
+});
+
+function generateRandomPhoneNumber() {
```

Comment:
Would this function be helpful in other places? Wondering if it can be moved to a shared file. Same for the function below.

Reply:
Sounds good, will look for a better place to have this method, but will not do it for this Test PR only for Staging branch